### ISSUE TITLE
medaka model for Guppy 4.0

### ISSUE THREAD

**Issue by osilander** [nanoporetech/medaka#181]
Which model is best for Guppy 4.0 (4.0.11+f1071ce)

As far as I can tell, for MinION 9.4.1 high accuracy data, medaka 1.0.3 has:
r941_min_high_g303
r941_min_high_g330
r941_min_high_g340_rle
r941_min_high_g344
r941_min_high_g351
r941_min_high_g360

Should I try g360?

**Comment by Kirk3gaard** [comment]
Any news on an ETA for medaka models for guppy v4?

**Comment by Kirk3gaard** [comment]
https://twitter.com/iiSeymour/status/1287709384609587200

**Comment by iiSeymour** [comment]
Hey @osilander 

Yes, only `template_r9.4.1_450bps_hac_prom.jsn` changed from 3.6 so please do use `g360`.

@mwykes can answer specifically on the next release which will cover `r941_hac_prom` model.

**Comment by Psy-Fer** [comment]
Ahh this answers the question for me too. Cheers!

**Comment by wufabai** [comment]
Hi @iiSeymour @mwykes , my sequencing provider told me that they used Guppy 4.0.11 option -hac for basecalling of the promethION run, but when I tried to use r941_prom_hac_g360 for medaka_concensus it says the model does not exist. May I ask if I should use an alternative model?
Also, I am trying to location a list of current models somewhere but I failed to find them. Could you point me towards it?
Thank you so much in advance!

**Comment by Psy-Fer** [comment]
I might be able to answer that one quickly.

You can see the list of models in the --help command. 

Also make sure you have an up to date version so it includes the later models. 

Hope this helps

**Comment by wufabai** [comment]
Thank you @Psy-Fer ! Appears that the model only showed up when I used medaka_concensus -h.
I just installed medaka a couple days ago and it is up to date. However, I cannot deduce which version of model to use for promethION & -hac option. Here are the models in the list:
r103_min_high_g345, r103_min_high_g360, r103_prom_high_g360, r103_prom_snp_g3210, r103_prom_variant_g3210, r10_min_high_g303, r10_min_high_g340, r941_min_fast_g303, r941_min_high_g303, r941_min_high_g330, r941_min_high_g340_rle, r941_min_high_g344, r941_min_high_g351, r941_min_high_g360, r941_prom_fast_g303, r941_prom_high_g303, r941_prom_high_g330, r941_prom_high_g344, r941_prom_high_g360, r941_prom_snp_g303, r941_prom_snp_g322, r941_prom_snp_g360, r941_prom_variant_g303, r941_prom_variant_g322, r941_prom_variant_g360.

**Comment by Psy-Fer** [comment]
It would be `r941_prom_high_g360`
However that is for the 3.6 guppy version, and the basecalling model changed in the 4.0+ versions.
So for the right model, you will have to either.
1. wait for the new 4.0+ model to come out with the next release of medaka

or

2. re-basecall with a 3.6 guppy, then use `r941_prom_high_g360`

**Comment by wufabai** [comment]
Thank you @Psy-Fer . I tried the r941_prom_high_g360 option, but somehow it ended up producing no fasta file in the folder after medaka_concensus run, despite showing no error. I will wait for the 4.0+ models to come out.

**Comment by Silverfoxcome** [comment]
> It would be `r941_prom_high_g360`
> However that is for the 3.6 guppy version, and the basecalling model changed in the 4.0+ versions.
> So for the right model, you will have to either.
> 
> 1. wait for the new 4.0+ model to come out with the next release of medaka
> 
> or
> 
> 1. re-basecall with a 3.6 guppy, then use `r941_prom_high_g360`

Hi! 

A question please :)

I'm using the accession SRX5632308 from the SRA database. It comes from Oxford Nanopore PromethION and was uploaded in 2019. That's is all I know about it. I want to polish my assemblies and, due to this being from 2019, I was thinking of using `r941_prom_hac_g303` or `r941_prom_high_g303`. But I'm not sure. Please, can you give me some pointers? 

Thanks a lot in advance.

**Comment by cjw85** [comment]
We. have just released v1.1.0 on. PyPI, it contains a PromethION model for Guppy 4.0.11.

**Comment by dcopetti** [comment]
Hello,
I tried installing the newest Medaka version to have the models for Guppy 4.0.11 and I get this error:
```
    Running setup.py install for edlib ... error
    ERROR: Command errored out with exit status 1:
     command: /home/copettid/miniconda2/envs/medaka_pip/bin/python3.7 -u -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/tmp/pip-install-5iisgb_q/edlib/setup.py'"'"'; __file__='"'"'/tmp/pip-install-5iisgb_q/edlib/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' install --record /tmp/pip-record-s321c91e/install-record.txt --single-version-externally-managed --compile --install-headers /home/copettid/miniconda2/envs/medaka_pip/include/python3.7m/edlib
         cwd: /tmp/pip-install-5iisgb_q/edlib/
    Complete output (11 lines):
    running install
    running build
    running build_ext
    building 'edlib' extension
    creating build
    creating build/temp.linux-x86_64-3.7
    creating build/temp.linux-x86_64-3.7/edlib
    creating build/temp.linux-x86_64-3.7/edlib/src
    gcc -pthread -B /home/copettid/miniconda2/envs/medaka_pip/compiler_compat -Wl,--sysroot=/ -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -Wstrict-prototypes -fPIC -Iedlib/include -I/home/copettid/miniconda2/envs/medaka_pip/include/python3.7m -c edlib.bycython.cpp -o build/temp.linux-x86_64-3.7/edlib.bycython.o -O3 -std=c++11
    gcc: error trying to exec 'cc1plus': execvp: No such file or directory
    error: command 'gcc' failed with exit status 1
    ----------------------------------------
ERROR: Command errored out with exit status 1: /home/copettid/miniconda2/envs/medaka_pip/bin/python3.7 -u -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/tmp/pip-install-5iisgb_q/edlib/setup.py'"'"'; __file__='"'"'/tmp/pip-install-5iisgb_q/edlib/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' install --record /tmp/pip-record-s321c91e/install-record.txt --single-version-externally-managed --compile --install-headers /home/copettid/miniconda2/envs/medaka_pip/include/python3.7m/edlib Check the logs for full command output.
```
this is just the final error, there were others before too.
I have python 3.7, is it a matter of the version?
Thanks!
